### PR TITLE
Feat/auto slippage

### PR DESCRIPTION
**Pull request by erwin-huang**

### CODE REVIEW

**Comment by erwin-huang** [comment]
To update `@koda-finance/summitswap-sdk` using `feat/auto-slippage` branch

We need to update package.json dependencies
`"@koda-finance/summitswap-sdk": "git+https://github.com/Koda-Finance/summitswap-sdk.git#feat/tokens-slippage-tolerance",`

And build `summitswap-sdk` and copy the dist to `summitswap-interface` path `./node-modules/@koda-finance/summitswap-sdk`

**Comment by erwin-huang** [comment]
Update: I pushed the `summitswap-sdk/dist` for testing purpose

File: src/pages/Swap/index.tsx [hunk 1]
```diff
@@ -177,6 +177,20 @@ const Swap: React.FC<IProps> = ({ isLanding }) => {
...
+    const buySlippageTolerance = (currencies[Field.OUTPUT] as Token).buySlippageTolerance || 0
+    const _allowedSlippage = sellSlippageTolerance > buySlippageTolerance ? sellSlippageTolerance : buySlippageTolerance
+
+    if (_allowedSlippage > 0) {
+      setAllowedSlippage(_allowedSlippage * 100)
+    }
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [currencies[Field.INPUT], currencies[Field.OUTPUT]])
+
```
Comment (LukeSamkharadze) [inline src/pages/Swap/index.tsx:193]: @erwin-huang what happens if first this effect sets the slippage, but later I choose another swap thats needs no slippage?

Comment (LukeSamkharadze) [inline src/pages/Swap/index.tsx:193]: @erwin-huang how do you handle default slippage?

Comment (LukeSamkharadze) [inline src/pages/Swap/index.tsx:193]: @erwin-huang how do you handle tolerance when there is a route of currencies?

For ex `BNB` -> `CMCC` -> `KODA`

Should we set max slippage tolerance of those tokens? (Currently you are just setting max between first and last)

Comment (erwin-huang) [reply src/pages/Swap/index.tsx]: Updated:
- I have added the default slippage, 0.8.
-  If there is a route of currencies, I will use the highest slippage that we will need for the swap.

**Review by LukeSamkharadze — APPROVED** [review]

**Pull request merged**